src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/ETagTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests
+{
+    public class ETagTests
+    {
+        [Test]
+        public void StringRoundtrips()
+        {
+            var s = "tag";
+            var eTag = new ETag(s);
+            Assert.AreSame(s, eTag.ToString());
+        }
+
+        [Test]
+        public void ComparisonOperators()
```

Comment:
I'd suggest breaking this down into several tests.   You're testing several different things under a broader category of `comparisons`.    While its convenient when authoring the test, failures aren't as clear due to the number of asserts that are only loosely related.   Each type of comparison, in my opinion, should have it's own dedicated test.

Reply:
Will do.